pages/customize/map.mdx

```diff
@@ -0,0 +1,122 @@
+import { Em, Heading, Section, Strong, Link, Box } from "@radix-ui/themes";
+
+# Map
+
+The Map component enhances Canopy sites with optional geospatial navigation, allowing users to explore the underlying
+collection through an interactive map. When activated, this component displays a map featuring markers representing each
+point Feature within the navPlace properties found throughout the manifests of the overarching collection.
+
+![Canopy Map Example](/docs/map-example.png)
+
+<Section size="2">
+
+  <Heading weight="medium">
+    What are <Strong>navPlace</Strong> Properties
+  </Heading>
```

Comment:
Let's not use Heading in here and keep it Markdown so that Nextra can properly build the pages table of contents.

This would just be `## What are _navPlace_ Properties` or something like that.